python/plasma/test/test.py

```diff
@@ -797,6 +797,80 @@ def test_stresstest(self):
 
     print("it took", b, "seconds to put and transfer the objects")
 
+class TestPlasmaManagerRecovery(unittest.TestCase):
+
+  def setUp(self):
+    # Start a Plasma store.
+    self.store_name, self.p2 = plasma.start_plasma_store(use_valgrind=USE_VALGRIND)
+    # Start a Redis server.
+    redis_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "../../core/src/common/thirdparty/redis/src/redis-server")
```

Comment:
can you use ray.services.start_redis here? It already does the retries correctly and loads the modules